test_programs/execution_success/bitshifts/src/main.nr

```diff
@@ -0,0 +1,32 @@
+// Prover.toml:
+// u8 = 100
+// positive_i8 = 100
+// negative_i8 = -100
+// seven = 7
+
+fn main(u8: u8, positive_i8: i8, negative_i8: i8, seven: u8) {
```

Comment:
uhhh, we allow `u8` as a variable name?

Reply:
Yes, like in Rust. That was made possible when I changed primitive names to not be keywords.